### DISCUSSION TITLE
How to get a list of gifts for a product in the OrderResource

### DISCUSSION THREAD

**Discussion by yablochnyi** [filamentphp/filament#5486] [Help]
Good afternoon! I have an order resource. In it I get a list of products. I have a table in the database with gifts, it is associated with products. How can I check the  products in the order resource, whether gifts are assigned to them, if they are, then display them in the order

**Comment by danharrin** [comment]
https://filamentphp.com/docs/2.x/admin/resources/relation-managers

**Reply by yablochnyi** [reply]
Without relation-managers, I can't get gifts in the select?

**Reply by danharrin** [reply]
https://filamentphp.com/docs/2.x/forms/fields#populating-automatically-from-a-relationship

**Reply by yablochnyi** [reply]
It's a little different. the resource order has not relation with the gifts table. In the resource order I receive the products
```
Forms\Components\Repeater::make('orderItems')
                                    ->label('заказу')
                                    ->relationship()
                                    ->schema([
                                        Forms\Components\Select::make('product_id')
                                            ->label('Продукт')
                                            ->options(Product::query()->pluck('name', 'id'))
                                            ->required()
                                            ->reactive()
                                            ->afterStateUpdated(fn($state, callable $set) => $set('price', Product::find($state)?->price ?? 0))
                                            ->columnSpan([
                                                'md' => 5,
                                            ]),
```
Next, I just want to send a request to the database, Check if the products from the order have gifts, and if so, display them in a multiselect. here is my test controller. But I don't know how to implement it in filament
```
public function test()
    {

        $gifts = GiftForProduct::query()->where('product_id', $product->id)->with('giftForProduct')->get();
        foreach ($gifts as $gift)
        {
            dd($gift->giftForProduct->name);
        }

    }
```
